src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -51,11 +54,6 @@ internal sealed class VariableState
         /// </summary>
         private readonly PooledDictionary<Symbol, TypeSymbolWithAnnotations> _variableTypes = PooledDictionary<Symbol, TypeSymbolWithAnnotations>.GetInstance();
 
-        /// <summary>
-        /// The current source assembly.
-        /// </summary>
-        private readonly SourceAssemblySymbol _sourceAssembly;
```

Comment:
>_sourceAssembly [](start = 46, length = 15)

Probably good to include this change with next opportune NullableWalker change into `master`.

Reply:
Yes, added to https://github.com/dotnet/roslyn/pull/31703.

Reply:
Yes, added to [https://github.com/dotnet/roslyn/pull/31703](https://github.com/dotnet/roslyn/pull/31703).

---
In reply to: [241227466](https://github.com/dotnet/roslyn/pull/31719#discussion_r241227466) [](ancestors = 241227466)